File: Java/Search/LinearSearch.java
package Search;

public class LinearSearch {
    public static int LinearSearch(int[] in, int Key) {
        for (int i = 0; i < in.length; i++) {
            if (Key == in[i])
                return i;
        }
        return -1;
    }

    public static void main(String[] args) {
        int[] theArray = { 1, 2, 34, 5, 25, 442, 1, 98, 225, 8 };
        System.out.println("The index number of 25 is " + LinearSearch(theArray, 25));
        System.out.println("The index number of 99 is " + LinearSearch(theArray, 99));
    }
}

File: Python/Search/LinearSearch.py
def LinearSearch(array, key):
    for index, theValue in enumerate(array):
        if(array[index] == key):
            return index
    return -1


def main():
    theArray = [2, 5, 25, 6, 8, 9, 1, 0, 6, 8, 4, 5, 66, 548, 889]
    print("The index of the 25 is: ", LinearSearch(theArray, 25))
    print("The index of the 91 is: ", LinearSearch(theArray, 91))


if __name__ == "__main__":
    main()
